### ISSUE TITLE
两机8卡，读取数据时hang住

### ISSUE THREAD

**Issue by Dorothylyly** [OpenRLHF/OpenRLHF#898]
<img width="871" alt="Image" src="https://github.com/user-attachments/assets/5b53a1e4-0f47-4890-a078-0801457c70fb" />
两机8卡，读取数据时hang住
ray start --head --node-ip-address 10.205.128.117 --port 6379 --num-gpus 8

ray job submit --address="http://127.0.0.1:8265" \
  --runtime-env-json='{"working_dir": "./"}' \
  -- python3 -m openrlhf.cli.train_ppo_ray \
  --ref_num_nodes 0 \
  --ref_num_gpus_per_node 0 \
  --critic_num_nodes 0 \
  --critic_num_gpus_per_node 0 \
  --actor_num_nodes 1 \
  --actor_num_gpus_per_node 8 \
  --vllm_num_engines 8 \
  --vllm_tensor_parallel_size 1 \
  --pretrain /path/to/file \
  --remote_rm_url reward_func_revisewformat.py \
  --save_path /path/to/file \
  --save_steps 150 \
  --ckpt_path /path/to/file \
  --save_hf_ckpt \
  --micro_train_batch_size 1 \
  --train_batch_size 128 \
  --micro_rollout_batch_size 1 \
  --rollout_batch_size 128 \
  --max_samples 1000000 \
  --max_ckpt_num 1 \
  --max_epochs 1 \
  --num_episodes 10 \
  --prompt_max_len 2048 \
  --generate_max_len 2048 \
  --zero_stage 3 \
  --bf16 \
  --actor_learning_rate 1e-6 \
  --advantage_estimator reinforce \
  --prompt_data json@/path/to/file \
  --input_key input_key \
  --label_key label_key \
  --apply_chat_template \
  --normalize_reward \
  --packing_samples \
  --adam_offload \
  --init_kl_coef 0 \
  --flash_attn \
  --gradient_checkpointing \
  --use_wandb 123

**Comment by Dorothylyly** [comment]
一直hang在99%
